packages/nextjs/src/common/wrapGenerationFunctionWithSentry.ts

```diff
@@ -68,7 +68,6 @@ export function wrapGenerationFunctionWithSentry<F extends (...args: any[]) => a
           {
             op: 'function.nextjs',
             name: `${componentType}.${generationFunctionIdentifier} (${componentRoute})`,
-            data,
```

Comment:
This looked sus but I saw that we also set it as extra on the scope.

Reply:
jup, that's why I removed it, this should remain the same on the event as far as I can tell! (it is also not flat so cannot be attributes directly, we'd need to flatten this but I don't think that's worth it here)